cmake/modules/packages/FindQB3.cmake

```diff
@@ -0,0 +1,60 @@
+
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying file COPYING-CMAKE-SCRIPTS or
+# https://cmake.org/licensing for details.
+
+#[=======================================================================[.rst:
+FindQB3
+--------
+
+Find the QB3 library
+
+QB3 library exports cmake config files
+
+IMPORTED targets
+^^^^^^^^^^^^^^^^
+
+This module defines the following 
+:prop_tgt:`IMPORTED` target: ``QB3::QB3``
+
+Result variables
+^^^^^^^^^^^^^^^^
+
+This module will set the following variables if found:
+
+``QB3_INCLUDE_DIR`` - where to find QB3 public headers
+``QB3_LIBRARY`` - the library to link against
+``QB3_FOUND`` - TRUE if found
+
+#]=======================================================================]
+
+find_package(libQB3 CONFIG)
```

Comment:
If upstream provides a CMake config file, that package name and its targets shall be used directly, and this `FindQB3.cmake` is obsolete.

Reply:
I think I got it.